src/components/DataCollector/createConfigValidators.js

```diff
@@ -27,6 +27,11 @@ export default () => {
     clickCollectionEnabled: {
       defaultValue: true,
       validate: boolean()
+    },
+    downloadLinkQualifier: {
+      defaultValue:
+        "\\.(exe|zip|wav|mp3|mov|mpg|avi|wmv|pdf|doc|docx|xls|xlsx|ppt|pptx)$",
+      validate: string().regexp()
     }
```

Comment:
Do we have plans to support linkInternalFilters like in AppMeasurement?

Reply:
I think we can leave that one out until we have a valid use-case.